tests/unit/amo/sagas/test_home.js

```diff
@@ -141,7 +141,75 @@ describe(__filename, () => {
       expect(loadAction).toEqual(expectedLoadAction);
     });
 
-    it('loads a null for a collection that cannot be fetched', async () => {
+    it('loads a null for a collection that returns a 401', async () => {
+      const state = sagaTester.getState();
```

Comment:
I'd like to try this new Jest 23 feature called "table driven tests" 🔥: https://facebook.github.io/jest/blog/#jest-each. (Documentation: https://facebook.github.io/jest/docs/en/api.html#testeachtable-name-fn)

We could write this test case that way:

```js
it.each([401, 404])('loads a null for a collection that returns a %d', async (status) => {
  // ...
});
```

WDYT? This patch is small enough to give it a try I'd say. I think we'll have to upgrade to Jest 23 though :)

Reply:
Ok, I've updated the test to use `it.each` and it works well. Thanks for the tip. This now depends on PR #5300 landing first.